modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -156,7 +156,7 @@ BYOC with customer-managed VPC::
 | Control plane | &#10003; |   
 | Access controls and audit | &#10003; | &#10003;  
 | Managed disaster recovery |  |  &#10003; 
-| Observability and monitoring (SLOs, SLIs, tracing, alerting, runbooks) | &#10003; | &#10003;  
+| Observability and monitoring (SLOs, SLIs, tracing, alerting, runbooks) | &#10003; | &#10003; (for VPC components managed by customer) 
```

Comment:
Suggestion, non-blocking: in BYOVPC the customer also supplies the cloud storage bucket, so perhaps `for VPC components and cloud storage buckets managed by customer` ?

Reply:
Thanks @gavinheavyside. I updated to: 

✓ (for VPC components and cloud storage buckets/containers managed by customer)